nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/test/java/org/apache/nifi/processors/poi/ConvertExcelToCSVProcessorTest.java

```diff
@@ -187,6 +213,34 @@ public void testSkipColumns() throws Exception {
                 "9.8765E+08,\n");
     }
 
+    @Test
+    public void testSkipColumnsWithEL() throws Exception {
+        testRunner.enqueue(new File("src/test/resources/dataformatting.xlsx").toPath());
```

Comment:
To test the EL evaluation againt attributes of the flow file, you can pass a Map<String, String> representing the flow file attributes in the enqueue() method.

Reply:
@pvillard31 do you want me to re-write the tests into a single method? initially there were separate methods for testing the skipRows and skipColumns, I just maintained the same format.